modules/mobile/app/models/mobile/v0/adapters/vaos_v2_appointments.rb

```diff
@@ -154,8 +160,8 @@ def cancel_id(appointment_hash)
           appointment_hash[:id]
         end
 
-        def type_of_care(service_type, type)
-          return nil if service_type.nil? || va?(type)
```

Comment:
Also solves a preexisting issue that type of care was not showing for va type appointments.